### ISSUE TITLE
Estimate cloud costs for 2023

### ISSUE THREAD

**Issue by pnorman** [openstreetmap/operations#788]
The cloud costs are the only recurring costs that vary significantly, so they should be worked out separately from other recurring costs as they require future planning.

**Comment by Firefishy** [comment]
Need to add Azure costs too, Azure is used at the moment for Translation services for community.osm.org
Current costs are expected to be £26 per month. Cost will increase as adoption of community.osm.org site grows.

**Comment by pnorman** [comment]
I've budgeted Azure under subscriptions since it's so small compared to other hosting costs, and it's used for one specific service.

**Comment by pnorman** [comment]
So, I had a browse through AWS cost explorer. There's a few things that make budgeting hard.

1. Since August 12th, #727 cut our AWS costs in half. This is, of course, a good thing, but it means we're restricted to 3.5 months of data we can use to forecast.
2. Costs consist of a predictable baseline with spikes for one-off batch operations. These batch operations might tasks like back-populating log tables, copying new files to S3, or ad-hoc of tile logs.
  ![image](https://user-images.githubusercontent.com/1190866/208026388-a05c55c3-dc4f-478a-8641-593b69eb772e.png)
3. #698 is not yet done - this should reduce our long-term costs as we will retain less data, but will shift some costs around in unknown ways.
4. For some reason, we've been getting credits since Oct 1st equal to our spend. We haven't had any bills to pay since the September bill which came in on October 2nd. This is **excluding** the render account. Because I don't know the source of this, I'm excluding these credits from the cost analysis.
5. Part of our S3 usage is backups, which we do not access on a regular basis. If we ever needed them, we would have significant bandwidth costs to download them.

Given all of these constraints, I fit the weekly data to a curve and got a cost of 26100 USD. This matches what I get for monthly data. If I used a linear fit to the last three months, I got 21k USD.
